### PR TITLE
Refactor mobile sessions endpoint

### PR DESCRIPTION
**Pull request by 3v0k4**
- Use form object
- New route
- Integration test

### CODE REVIEW

File: app/services/api/to_mobile_sessions_array.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  def limit
+    data[:limit]
+  end
+
+  def offset
+    data[:offset]
+  end
+end
```
Comment (AkkarinForest) [inline app/services/api/to_mobile_sessions_array.rb]: This file looks very similar to `to_dormant_sessions_array`, maybe you can extract some duplication?

File: app/controllers/api/measurement_sessions_controller.rb [hunk 2]
```diff
@@ -27,17 +27,6 @@ class MeasurementSessionsController < BaseController
 
     respond_to :json
 
-    def index
-      data = decoded_query_data(params[:q])
-      INT_Q_ATTRS.each { |key| data[key] = data[key].to_i if data.key?(key) }
```
Comment (AkkarinForest) [inline app/controllers/api/measurement_sessions_controller.rb]: You can also delete the `INT_Q_ATTRS ` array on line 21.

File: app/services/api/to_mobile_sessions_array.rb (hunk 1 above)
Comment (3v0k4) [reply app/services/api/to_mobile_sessions_array.rb]: Prolly the pagination is similar. I believe that the duplication is not that bad. Do you have any proposals?

File: app/controllers/api/measurement_sessions_controller.rb (hunk 2 above)
Comment (3v0k4) [reply app/controllers/api/measurement_sessions_controller.rb]: Good catch!

**Pull request merged**